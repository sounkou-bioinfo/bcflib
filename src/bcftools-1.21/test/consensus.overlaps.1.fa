>1
CGGGACAC
>2
CGGGACAC
