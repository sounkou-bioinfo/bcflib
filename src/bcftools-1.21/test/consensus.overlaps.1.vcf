##fileformat=VCFv4.2
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##contig=<ID=1>
##contig=<ID=2>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	A	B
1	2	.	GGGAC	G,<NON_REF>	649.73	.	.	GT	1/1	1/1
1	6	.	CA	C,<NON_REF>	536.73	.	.	GT	0/0	1/1
2	2	.	GGGAC	G,<NON_REF>	649.73	.	.	GT	1/1	1/1
2	6	.	CA	C,<NON_REF>	536.73	.	.	GT	0/0	1/1
2	7	.	A	AT,<NON_REF>	649.73	.	.	GT	1/1	1/1
