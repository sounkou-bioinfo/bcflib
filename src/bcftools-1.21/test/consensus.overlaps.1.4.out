>1
NGN
>2
NGTN
